cmd/algorand-indexer/main.go

```diff
@@ -97,6 +99,11 @@ func globalIndexerDb(opts *idb.IndexerDbOptions) idb.IndexerDb {
 }
 
 func init() {
+	logger = log.New()
+	logger.SetFormatter(&log.JSONFormatter{})
+	logger.SetOutput(os.Stdout)
+	logger.SetLevel(log.InfoLevel)
```

Comment:
Consider making it a flag (Either in this PR, or ticket it for another one)

Reply:
Definitely. This is a baby step towards #179